example/ios/example/Info.plist

```diff
@@ -38,20 +38,15 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-		<string>facebook</string>
-		<string>whatsapp</string>
-		<string>mailto</string>
-	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
-	<key>NSPhotoLibraryAddUsageDescription</key>
```

Comment:
not sure if the example uses this permission or not (I didn't cross-check App.js) but on iOS if you try to access something without a permission description you just crash immediatley